scss/_settings_global.scss

```diff
@@ -1,120 +1,120 @@
-/// standard colors
+// standard colors
 
-/// assets database path
-/// usage: background: url(#{$asset-path}img//backgrounds/background.jpg) no-repeat 0 0;
+// assets database path
+// usage: background: url(#{$asset-path}img//backgrounds/background.jpg) no-repeat 0 0;
 $asset-path:            'https://assets.ubuntu.com/sites/ubuntu/latest/u/' !default;
 $asset-server:          'https://assets.ubuntu.com/v1/' !default;
 
-/// Base font family
-/// default: Ubuntu, Arial, 'libra sans', sans-serif
+// Base font family
+// default: Ubuntu, Arial, 'libra sans', sans-serif
 $base-font-family:      "Ubuntu, Arial, 'libra sans', sans-serif" !default;
 
-/// Base font size
+// Base font size
 // default: 16px
 $base-font-size:        16px !default;
 
-/// Heading font family
-/// default: $base-font-family
+// Heading font family
+// default: $base-font-family
 $heading-font-family:   $base-font-family !default;
 
-/// transparent to use throughout the site
+// transparent to use throughout the site
 $transparent:           transparent !default;
```

Comment:
Duplicated variables. You moved them to a colours settings file. These should be removed